clap_derive/tests/subcommands.rs

```diff
@@ -165,6 +165,8 @@ fn test_tuple_commands() {
 fn external_subcommand() {
     #[derive(Debug, PartialEq, Clap)]
     struct Opt {
+        #[clap(global = true, long)]
+        global: bool,
```

Comment:
Try writing a separate test case instead of combining it with this external subcommand test.